test_vector_handlers/src/awses_test_vectors/internal/util.py

```diff
@@ -19,7 +19,11 @@
 
 import six
 from attr import Attribute  # noqa pylint: disable=unused-import
-from aws_encryption_sdk.identifiers import AlgorithmSuite
+
```

Comment:
They're in the same repository; they're different packages specifically so that we can test with different versions of the encryption SDK. I went with the same repository just because it didn't seem like there was a strong reason to have a lot of separate repositories with strongly related stuff in them.